### ISSUE TITLE
Italic fonts

### ISSUE THREAD

**Issue by andreas-enge** [ReScience/ReScience#97]
Hello,

I would like to propose the attached patch to rescience.cls. It enables italic fonts in more places (the abstract, for instance).

Andreas
[rescience-cls.txt](https://github.com/ReScience/ReScience/files/5607177/rescience-cls.txt)

**Comment by andreas-enge** [comment]
Actually I realise that only the first hunk of the patch still applies, but this is also the only one that I have tested (using some italic text in an abstract).

**Comment by rougier** [comment]
@andreas-enge Best would be to make a pull request at https://github.com/ReScience/template/blob/master/rescience.cls. But from your patch, you're only modifying Source Code Pro (for listing) and Roboto Condensed (for section titles). If you mean the abstract, maybe this won't work.

**Comment by andreas-enge** [comment]
Am Sat, Nov 28, 2020 at 02:20:32AM -0800 schrieb Nicolas P. Rougier:
> @andreas-enge Best would be to make a pull request at https://github.com/
> ReScience/template/blob/master/rescience.cls. But from your patch, you're only
> modifying Source Code Pro (for listing) and Roboto Condensed (for section
> titles). If you mean the abstract, maybe this won't work.

I made a pull request with only the change for Roboto Condensed, which is
apparently used in the abstract - I checked that it avoids this warning
   LaTeX Font Warning: Font shape `TU/RobotoCondensed(0)/m/it' undefined
   (Font)              using `TU/RobotoCondensed(0)/m/n' instead on input line 85.
for my 10-years-challenge article and prints italic content in the abstract.

Andreas
